<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPDU_OutputSwItemWid</class>
 <widget class="QWidget" name="MPDU_OutputSwItemWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输出位</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="openRadio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="closeRadio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
